config/upcloud/group_vars/k8s_cluster/ck8s-k8s-cluster-upcloud.yaml

```diff
@@ -0,0 +1,30 @@
+etcd_deployment_type: kubeadm
+
+## Repo for UpClouds csi-driver: https://github.com/UpCloudLtd/upcloud-csi
+## To use UpClouds CSI plugin to provision volumes set this value to true
+## Remember to set UPCLOUD_USERNAME and UPCLOUD_PASSWORD
+upcloud_csi_enabled: true
+upcloud_csi_enable_volume_snapshot: true
+
+#upcloud_csi_controller_replicas: 1
+## Override used image tags
+# upcloud_csi_provisioner_image_tag: "v3.1.0"
+# upcloud_csi_attacher_image_tag: "v3.4.0"
+# upcloud_csi_resizer_image_tag: "v1.4.0"
+# upcloud_csi_plugin_image_tag: "alpha"
+# upcloud_csi_node_image_tag: "v2.5.0"
+upcloud_tolerations:
+  - key: elastisys.io/node-type
+    operator: Exists
+    effect: NoSchedule
+
+## Storage class options
+expand_persistent_volumes: true
+persistent_volumes_enabled: true
+storage_classes:
+  - name: standard
+    is_default: true
+
+supplementary_addresses_in_ssl_keys: [<loadbalancer-domain>, "kube.ops.CHANGE-ME-BASE-DOMAIN"]
+
+kubeconfig_localhost_ansible_host: true
```

Comment:
These are commented out in the OpenStack config: https://github.com/elastisys/compliantkubernetes-kubespray/blob/dcd2c00a724c33f9279f37c97f0762bd6a06314f/config/openstack/group_vars/k8s_cluster/ck8s-k8s-cluster-openstack.yaml#L42-L44

Why is it different here?

Reply:
I did a quick check in our internal templates and in it, both openstack and upcloud have this config uncommented. I'll uncomment the openstack condig here.